regression in lru

The code now fails to build with Cython 3.1.2 on mingw, it looks like 2f019f963d4eb3ef89cec614145aa55b85501c8b broke something:
```
C:/msys64/mingw64/include/python3.12/cpython/odictobject.h:26:43: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'PyODictObject *' {aka 'struct _odictobject *'}
   26 | PyAPI_FUNC(int) PyODict_SetItem(PyObject *od, PyObject *key, PyObject *item);
      |                                 ~~~~~~~~~~^~
winloop/loop.c:79088:34: error: passing argument 1 of 'PyODict_SetItem' from incompatible pointer type [-Wincompatible-pointer-types]
79088 |     __pyx_t_3 = PyODict_SetItem(((PyODictObject *)__pyx_t_2), __pyx_v_key, __pyx_v_o); if (unlikely(__pyx_t_3 == ((int)-1))) __PYX_ERR(13, 163, __pyx_L1_error)
      |                                 ~^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                  |
      |                                  PyODictObject * {aka struct _odictobject *}
C:/msys64/mingw64/include/python3.12/cpython/odictobject.h:26:43: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'PyODictObject *' {aka 'struct _odictobject *'}
   26 | PyAPI_FUNC(int) PyODict_SetItem(PyObject *od, PyObject *key, PyObject *item);
      |                                 ~~~~~~~~~~^~
winloop/loop.c: In function '__pyx_pf_7winloop_4loop_8LruCache_6__delitem__':
winloop/loop.c:79180:32: error: passing argument 1 of 'PyODict_DelItem' from incompatible pointer type [-Wincompatible-pointer-types]
79180 |   __pyx_t_2 = PyODict_DelItem(((PyODictObject *)__pyx_t_1), __pyx_v_key); if (unlikely(__pyx_t_2 == ((int)-1))) __PYX_ERR(13, 168, __pyx_L1_error)
      |                               ~^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                |
      |                                PyODictObject * {aka struct _odictobject *}
C:/msys64/mingw64/include/python3.12/cpython/odictobject.h:27:43: note: expected 'PyObject *' {aka 'struct _object *'} but argument is of type 'PyODictObject *' {aka 'struct _odictobject *'}
   27 | PyAPI_FUNC(int) PyODict_DelItem(PyObject *od, PyObject *key);
      |                                 ~~~~~~~~~~^~
error: command 'C:\\msys64\\mingw64\\bin/gcc.EXE' failed with exit code 1
```

--- a/winloop/lru.pyx
+++ b/winloop/lru.pyx
@@ -86,15 +86,19 @@ PyAPI_FUNC(int) PyODict_DelItem(PyObject *od, PyObject *key);
     ctypedef class _collections.OrderedDict [object PyODictObject, check_size ignore]:
         pass
 
+    # OrderedDict cannot be defined in any of it's C Methods due to GCC 
+    # SEE: https://github.com/Vizonex/Winloop/issues/64
+
     bint PyODict_Check(object op)
     bint PyODict_CheckExact(object op)
-    Py_ssize_t PyODict_SIZE(OrderedDict op)
-    int PyODict_Contains(OrderedDict op, object key) except -1
+    Py_ssize_t PyODict_SIZE(object op)
+    int PyODict_Contains(object op, object key) except -1
     OrderedDict PyODict_New()
-    int PyODict_SetItem(OrderedDict od, object key, object item) except -1
-    int PyODict_DelItem(OrderedDict od, object key) except -1
-    PyObject* PyODict_GetItem(OrderedDict od, object key)
-    object PyODict_GetItemWithError(OrderedDict od, object key)
+    
+    int PyODict_SetItem(object od, object key, object item) except -1
+    int PyODict_DelItem(object od, object key) except -1
+    PyObject* PyODict_GetItem(object od, object key)
+    object PyODict_GetItemWithError(object od, object key)
 
 
 @cython.final

